L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -186,6 +192,9 @@
         nTrack = cms.uint32(30),
         nIn = cms.uint32(20),
         nOut = cms.uint32(20),
+        vtxRes      = cms.double(0.333),
+        nVtx        = cms.uint32(1),    
+        etraVtx     = cms.bool(False),    

```

Comment:
same for `etraVtx`, which is also misspelled

Reply:
cleaned up.